VERSION

```diff
@@ -1 +1 @@
-0.26.4
+0.26.5
```

Comment:
Wasn't sure if just adding optional build arguments to a target required a version bump, but bumped anyway. I can revert this change if it is not necessary.